sphinx/templates/quickstart/Makefile_t

```diff
@@ -94,4 +95,3 @@ gettext:
 .PHONY: Makefile
 %: Makefile
 	$(SPHINXBUILD) -b "$@" $(ALLSPHINXOPTS) "$(BUILDDIR)/$@"
-
```

Comment:
Why removing this?

Reply:
I believe my editor was configured to strip more that 1 trailing white line at the end of the file. The file had two blank lines at the bottom. If this change is too noisy I can revert it.

Reply:
It seems our test case expects a carriage return before EOF. Please revert this line or update a test case.